internal/static/analysis.go

```diff
@@ -0,0 +1,71 @@
+package static
+
+import (
+	"go/constant"
+	"go/types"
+
+	"github.com/dogmatiq/configkit"
+	"github.com/dogmatiq/configkit/internal/entity"
+	"github.com/dogmatiq/configkit/internal/typename/gotypes"
+	"golang.org/x/tools/go/ssa"
+)
+
+// analyzeApplication analyzes the member of the package and populates the
+// collected datainto entity.Application.
+//
+// The second parameter is the type that actually implements
+// github.com/dogmatiq/dogma.Application interface.
+func analyzeApplication(m ssa.Member, typ types.Type) *entity.Application {
+	app := &entity.Application{
+		TypeNameValue: gotypes.NameOf(typ),
+	}
+
+	fn := m.Package().Prog.LookupMethod(
```

Comment:
Still got "feature envy" here... `m` is not actually what this function needs, it needs the `m.Package()` so we should just pass the package as a parameter directly.

Reply:
Sorry man, Github placed your comments into hidden conversations for some reason. I am addressing it now...

Reply:
This should be fixed in https://github.com/dogmatiq/configkit/pull/118/commits/91f3e61a4db7c1deac9948b1a4995d53a891cd53.